### PR TITLE
[7638] remove background_task tables

### PR DESCRIPTION
**Pull request by m4ra**
**Tasks**
- [x] PR name contains story or task reference
- [ ] Documentation (docs and inline)
- [ ] Tests (including n+1 and django_assert_num_queries where applicable)
- [x] Changelog

### CODE REVIEW

File: meinberlin/apps/contrib/migrations/0001_drop_background_task_table.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Generated by Django 3.2.20 on 2023-11-09 07:46
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = []
+
+    operations = [migrations.RunSQL("DROP TABLE IF EXISTS background_task")]
```
Comment (goapunk) [inline meinberlin/apps/contrib/migrations/0001_drop_background_task_table.py:9]: background tasks creates two tables: `background_task` and `background_task_completedtask`, do you want to keep the second one or is it missing ? :)

**Review by goapunk — CHANGES_REQUESTED** [review]
Nice, just a question about the second table

File: meinberlin/apps/contrib/migrations/0001_drop_background_task_table.py (hunk 1 above)
Comment (m4ra) [reply meinberlin/apps/contrib/migrations/0001_drop_background_task_table.py:9]: oh no I don't, I didn't add it, slipped me.

**Review by goapunk — APPROVED** [review]
LGTM, just one question I haven't really considered before: by deleting the table manually, if we ever want to add background-tasks back the database still thinks the tables exist in the migration history, so it won't create them again. Do you think this is a problem? Or should we just ignore it?

**Comment by m4ra** [comment]
@goapunk background tasks are not supported in Django 4, so we won't go back to these.

**Comment by goapunk** [comment]
> @goapunk background tasks are not supported in Django 4, so we won't go back to these.

yes, true, was more of a general thought that running custom sql will break the migration history.

**Comment by m4ra** [comment]
The migration is not dependent in any other though. So I don't think that is an issue. And the background tasks migrations come from the package, not from our own repo. Once the package is deleted, its history also is gone afaik

**Comment by m4ra** [comment]
Moreover, if the app label is not in the settings. django migrations won't look for these.

**Review by goapunk — APPROVED** [review]
cool

**Pull request merged**